package.json

```
@@ -39,11 +39,11 @@
     "sqlite3": "^3.0.8"
   },
   "peerDependencies": {
-    "knex": "> 0.8 < 0.96"
+    "knex": "> 0.8 < 2.0"
```

Sorry, I commented on the wrong side of the diff before. I just tested this and this bit excludes 2.0 as written. Changing to ‘> 0.8 <= 2.0’ works.

---

ah alright, strange it worked for us, anyway you got it
```suggestion
    "knex": "> 0.8 <= 2.0"
```